### PR TITLE
minor: remove excludes(related to checkstyle#10956)

### PR DESCRIPTION
**Pull request by nmancus1**
Related to comments at https://github.com/checkstyle/checkstyle/pull/10956#issuecomment-970332760

### CODE REVIEW

**Review by pbludov — APPROVED** [review]

File: checkstyle-tester/projects-to-test-on-for-github-action.properties [hunk 1]
```diff
@@ -3,31 +3,30 @@
 # Please note that bash comments works in this file
 
 # Few projects that delivers set of unusual Java constructions that shall be correctly handled by AST visitor
-# 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
-checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java
```
Comment (rnveach) [inline checkstyle-tester/projects-to-test-on-for-github-action.properties:7]: why remove non-compilable?

File: checkstyle-tester/projects-to-test-on-for-github-action.properties [hunk 2]
```diff
@@ -3,31 +3,30 @@
...
 spotbugs|git|https://github.com/spotbugs/spotbugs|3.1.2||
-pmd|git|https://github.com/pmd/pmd|pmd_releases/6.21.0|**/pmd/pmd-java/src/test/**/*,**/pmd/cpd/files/*
+pmd|git|https://github.com/pmd/pmd|pmd_releases/6.21.0|
 spoon|git|https://github.com/INRIA/spoon.git|spoon-core-8.0.0|
-lombok-ast|git|https://github.com/rzwitserloot/lombok.ast|v0.2|**/lombok-ast/test/**/*
+lombok-ast|git|https://github.com/rzwitserloot/lombok.ast|v0.2|
 
 spring-framework|git|https://github.com/spring-projects/spring-framework|v4.1.6.RELEASE||
-hibernate-orm|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|**/hibernate-orm/documentation/**/*
+hibernate-orm|git|https://github.com/hibernate/hibernate-orm|4.2.19.Final|
```
Comment (rnveach) [inline checkstyle-tester/projects-to-test-on-for-github-action.properties:18]: Why is it ok to remove this one?

**Review by rnveach — COMMENTED** [review]
Do we have any example run with these removals?

**Comment by nmancus1** [comment]
> Do we have any example run with these removals?

I can open a dummy PR in main repo if you would like.

These excluded files were always ignored. Now that we have merged https://github.com/checkstyle/checkstyle/pull/10956#issuecomment-970332760, we will honor these excluded files, so we should remove them to be consistent with previous diff report generation behavior.

**Review by romani — APPROVED** [review]
Ok to merge

**Comment by nmancus1** [comment]
Report generated before allowing excludes (2 days ago): https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/27d4f52_2021143110/reports/diff/index.html

Report using this config: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/ed72733_2021020602/reports/diff/index.html

Report using regular config: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/ed72733_2021024634/reports/diff/index.html

**Comment by rnveach** [comment]
@nmancus1 
> These excluded files were always ignored. Now that we have merged checkstyle/checkstyle#10956 (comment), we will honor these excluded files, so we should remove them to be consistent with previous diff report generation behavior (having no excluded files).

Were any of these used as part of no-exception runs? These lists did have a valid use as part of the no-exception runs to prevent processing invalid files for those. No exception runs do not keep going if there is an exception on an invalid file.

**Comment by nmancus1** [comment]
> Were any of these used as part of no-exception runs? 

````bash
➜  checkstyle git:(master) grep -R "projects-to-test-on-for-github-action.properties"
.github/workflows/diff_report.yml:  DEFAULT_PROJECTS_LINK: "https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/projects-to-test-on-for-github-action.properties"
➜  checkstyle git:(master) 

```

**Comment by rnveach** [comment]
Thanks, I missed the name of the file. So its only used for differences.

**Review by rnveach — APPROVED** [review]

**Pull request merged**